Button/Button.js

```diff
@@ -118,7 +121,8 @@ const ButtonBase = kind({
 	},
 
 	computed: {
-		className: ({highlighted, joinedPosition, selected, type, styler}) => styler.append(
+		className: ({backgroundOpacity, highlighted, joinedPosition, selected, type, styler}) => styler.append(
+			backgroundOpacity,
```

Comment:
If we're going to pass this through to styler, we should define the class in the CSS even if it is unvalued. Otherwise, styler will treat it as a global class and apply it as is which could have unexpected side effects for authors.

Reply:
Oh good point.